Missing include header file for struct tm in util.c when compiling on msvc 2017 x64

Missing include header file for struct tm in util.c when compiling on msvc 2017 x64.

>  C2224: “.tm_mon”的左侧必须具有结构/联合类型
2>C:\githubrepo\nats.c\src\util.c(1307): error C2224: “.tm_mday”的左侧必须具有结构/联合类型
2>C:\githubrepo\nats.c\src\util.c(1307): error C2224: “.tm_hour”的左侧必须具有结构/联合类型
1>C:\githubrepo\nats.c\src\util.c(1306): warning C4477: “sscanf”: 格式字符串“%4d”需要类型“int *”的参数，但可变参数 1 拥有了类型“char *”
1>C:\githubrepo\nats.c\src\util.c(1306): note: 请考虑在格式字符串中使用“%hhd”
2>C:\githubrepo\nats.c\src\util.c(1307): error C2224: “.tm_min”的左侧必须具有结构/联合类型
2>C:\githubrepo\nats.c\src\util.c(1307): error C2224: “.tm_sec”的左侧必须具有结构/联合类型
2>C:\githubrepo\nats.c\src\util.c(1306): warning C4477: “sscanf”: 格式字符串“%4d”需要类型“int *”的参数，但可变参数 1 拥有了类型“char *”
2>C:\githubrepo\nats.c\src\util.c(1306): note: 请考虑在格式字符串中使用“%hhd”
1>C:\githubrepo\nats.c\src\util.c(1306): warning C4473: “sscanf”: 没有为格式字符串传递足够的参数
1>C:\githubrepo\nats.c\src\util.c(1306): note: 占位符和其参数预计 9 可变参数，但提供的却是 3 参数
1>C:\githubrepo\nats.c\src\util.c(1306): note: 缺失的可变参数 4 为格式字符串“%2d”所需
1>C:\githubrepo\nats.c\src\util.c(1313): error C2224: “.tm_year”的左侧必须具有结构/联合类型
1>C:\githubrepo\nats.c\src\util.c(1314): error C2224: “.tm_mon”的左侧必须具有结构/联合类型
2>C:\githubrepo\nats.c\src\util.c(1306): warning C4473: “sscanf”: 没有为格式字符串传递足够的参数
2>C:\githubrepo\nats.c\src\util.c(1306): note: 占位符和其参数预计 9 可变参数，但提供的却是 3 参数
2>C:\githubrepo\nats.c\src\util.c(1306): note: 缺失的可变参数 4 为格式字符串“%2d”所需
2>C:\githubrepo\nats.c\src\util.c(1313): error C2224: “.tm_year”的左侧必须具有结构/联合类型
1>C:\githubrepo\nats.c\src\util.c(1315): error C2224: “.tm_isdst”的左侧必须具有结构/联合类型
1>C:\githubrepo\nats.c\src\util.c(1317): warning C4013: “_mkgmtime64”未定义；假设外部返回 int
1>C:\githubrepo\nats.c\src\util.c(1524): error C2079: “tp”使用未定义的 struct“tm”
1>C:\githubrepo\nats.c\src\util.c(1540): error C2027: 使用了未定义类型“tm”
1>C:\githubrepo\nats.c\src\util.c(1524): note: 参见“tm”的声明
1>C:\githubrepo\nats.c\src\util.c(1542): warning C4013: “_gmtime64_s”未定义；假设外部返回 int
1>C:\githubrepo\nats.c\src\util.c(1546): warning C4013: “strftime”未定义；假设外部返回 int
2>C:\githubrepo\nats.c\src\util.c(1314): error C2224: “.tm_mon”的左侧必须具有结构/联合类型
2>C:\githubrepo\nats.c\src\util.c(1315): error C2224: “.tm_isdst”的左侧必须具有结构/联合类型
2>C:\githubrepo\nats.c\src\util.c(1317): warning C4013: “_mkgmtime64”未定义；假设外部返回 int
2>C:\githubrepo\nats.c\src\util.c(1524): error C2079: “tp”使用未定义的 struct“tm”
2>C:\githubrepo\nats.c\src\util.c(1540): error C2027: 使用了未定义类型“tm”
2>C:\githubrepo\nats.c\src\util.c(1524): note: 参见“tm”的声明
2>C:\githubrepo\nats.c\src\util.c(1542): warning C4013: “_gmtime64_s”未定义；假设外部返回 int
2>C:\githubrepo\nats.c\src\util.c(1546): warning C4013: “strftime”未定义；假设外部返回 int
1>已完成生成项目“nats.vcxproj”的操作 - 失败。

Add  #include <time.h> to util.c fix this issue but I'm not sure this is a good fix.

--- a/src/util.c
+++ b/src/util.c
@@ -17,6 +17,7 @@
 #include <stdarg.h>
 #include <string.h>
 #include <ctype.h>
+#include <time.h>
 
 #include "util.h"
 #include "mem.h"

